docker-compose.yml

```diff
@@ -39,6 +40,8 @@ services:
       OAUTH2_PROXY_COOKIE_SECRET: ${OAUTH2_PROXY_COOKIE_SECRET}
       OAUTH2_PROXY_OIDC_ISSUER_URL: ${OAUTH2_PROXY_OIDC_ISSUER_URL}
       OAUTH2_PROXY_REDIRECT_URL: ${OAUTH2_PROXY_REDIRECT_URL}
+      # Temporary branch for brk2
+      REPO_BASE: "https://raw.githubusercontent.com/Amsterdam/amsterdam-schema/brk2_active_dataset"
```

Comment:
How is this used? Are we using everything from this branch now?

I would suggest only using this branch for BRK2, by explicitly declaring the branch to use in gobmodel.json when the branch is not master.

Reply:
Agreed